modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/ConfigurationAsmGenerator.java

```diff
@@ -2588,6 +2646,14 @@ private BytecodeBlock treatSourceForConstruct(
                 setField = setThisFieldCode(constructMtd, newValue, schemaFieldDef);
             }
 
+            if (isContainNameAnnotation(schemaField)) {
```

Comment:
This name just feels wrong. No one says "is contain", maybe rename it to "containsNameAnnotation"?